src/main/java/core/basesyntax/Node.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax;
+
+public class Node<T> {
+
+    private Node prev;
```

Comment:
The checklist advises against using class Objects. Consider using `Node<T>` instead of `Node` for the `prev` field.